README.md

```diff
@@ -55,6 +55,13 @@ tox -e behave features/unattached_commands.feature
 tox -e behave features/unattached_commands.feature:55
 ```
 
+When developing/debugging a new scenario:
+
+ 1. Add a `@wip` tag decorator on the scenario
+ 2. Run `tox -e behave -- -w`   # will only run @wip scenarios with verbose logs
+ 3. Optionally `import ipdb; ipdb.set_trace()`
```

Comment:
How does this fit in? Do I put this in my code? my test?

Reply:
Thanks for the review, force pushed changes to address this.